forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -431,6 +431,7 @@ def get(form_id: string, submission_id: string):
         """PDF generation and rendering method."""
         try:
             if auth.has_one_of_roles([REVIEWER_GROUP, CLIENT_GROUP]):
+                timezone = request.args["timezone"]
```

Comment:
KeyError if no "timezone" in `request.args`.
Pls make sure `return err.error, err.status_code` wont crash the server

Reply:
Okay will update